lib/application_responder.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationResponder < ActionController::Responder
+  include Responders::FlashResponder
+  include Responders::HttpCacheResponder
+
+  # Redirects resources to the collection path (index action) instead
+  # of the resource path (show action) for POST/PUT/DELETE requests.
+  # include Responders::CollectionResponder
+end
```

Comment:
FlashResponder
 # Responder to automatically set flash messages based on I18n API. It checks for
 # message based on the current action, but also allows defaults to be set, 

HttpCacheResponder
  # Set HTTP Last-Modified headers based on the given resource. It's used only
  # on API behavior (to_format) and is useful for a client to check in the server
  # if a resource changed after a specific date or not.
  #
  # This is not usually not used in html requests because pages contains a lot
  # information besides the resource information, as current_user, flash messages,
  # widgets... that are better handled with other strategies, as fragment caches and
  # the digest of the body.